bm6/bm6.c

```diff
@@ -23,41 +23,41 @@ static short int b12;
 
 short array[11];
 
-void bar ()
+void bm6(void)
 {
 
-  b1 = foo(1);
-  b2 = foo(1);
-  b3 = foo(1);
-  b4 = foo(1);
-  b5 = foo(1);
-  b6 = foo(1);
-  b7 = foo(1);
-  b8 = foo(1);
-  b9 = foo(1);
+  b1  = foo(1);
```

Comment:
Why the extra spaces?

Reply:
Oh I see; you are making the calls to `foo(1)` line up.

Reply:
Offtopic: with geany or notepad++ that's so convenient with the shift alt muliline cursor..but I guess you know that already.